### ISSUE TITLE
Consider creating a package with Spack

### ISSUE THREAD

**Issue by FrancescAlted** [Blosc/c-blosc2#67]
[Spack](https://spack.readthedocs.io/en/latest/index.html) looks like a good fit for creating and distributing binaries for c-blosc2.  Worth a try.

**Comment by psychocoderHPC** [comment]
@ax3l Since you are currently creating a spack package for blosc I like to point you to this open issue.

**Comment by ax3l** [comment]
I already created one and will mainline it soon.
The only thing I am struggling with is: https://github.com/inikep/lizard/issues/21

**Comment by ax3l** [comment]
Little note on spack: (public) binary mirrors will come soon (or one can roll one already), currently it's all source-code distribution.

**Comment by FrancescAlted** [comment]
Hi @ax3l .  Thanks for looking into this.  I am pretty impressed by how powerful spack looks like, and having binary repos would be the cherry in the cake.

Regarding support for lizard, why you don't use the internal sources in c-blosc2 for the time being?  The same could apply to the other codecs (specially lz4 and zstd).

**Comment by FrancescAlted** [comment]
Ok.  I see that you already enabled lizard build directly from c-blosc2 😊

**Comment by ax3l** [comment]
Yep, was wondering if the missing lizard decompress header is indeed a public header as reported upstream in https://github.com/inikep/lizard/issues/21 or if this a c-blosc2 issue using them?
Either way, I do see issues building tests and examples when building, even with an internal, lizard.

**Comment by FrancescAlted** [comment]
I see; cool!

And regarding the public header issue, I never had a problem with Lizard, and I used it in a pretty large combination of OS and compilers.  Hopefully you will figure out what's going on there.

**Comment by ax3l** [comment]
The upstream lizard issue is now fixed in https://github.com/inikep/lizard/issues/21#issuecomment-546266776 .

I created a spack package for it that also applies the patch in https://github.com/spack/spack/pull/13456 .
